apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -293,6 +307,43 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
         setCountryList(ReusableCommonUtils.getCountryList());
     }, []);
 
+    /**
+     * API call to get validation configurations.
+     */
+    const getConfigurations = (): void => {
+
+        fetchPasswordValidationConfig()
+            .then((response: ValidationFormInterface[]) => {
+                setUsernameConfig(getUsernameConfiguration(response));
+            })
+            .catch((error: any) => {
```

Comment:
Let's define the type for error other than `any`.
We should be able to use `IdentityAppsApiException` here.

Reply:
Addressed with 2255de611fd68c6c9f2029e0bfd1dbab1c90e8c3.